common/messaging/kafkaClient.go

```diff
@@ -30,6 +32,7 @@ type (
 	kafkaClient struct {
 		config *KafkaConfig
 		client *kafkaclient.Client
```

Comment:
I think all you need is to glide up and make this a non-pointer

Reply:
I will take care of it in a separate PR.